packages/components/src/draggable/index.tsx

```diff
@@ -82,6 +82,11 @@ export function Draggable( {
 		event.preventDefault();
 		cleanupRef.current();
 
+		// Blur the drag handle to clear the :active state.
+		if ( event.target instanceof HTMLElement ) {
```

Comment:
Since this element is not in an iframe, it's fine, but I find it generally better to explicitly get `HTMLElement` from `defaultView` to communicate what is expected.